composer.json

```diff
@@ -37,19 +37,21 @@
                 }
             }
         }
-    },
+    ],
     "require": {
         "ext-dom": "*",
         "composer/installers": "^1.10",
         "cweagans/composer-patches": "^1.6",
         "drupal/acquia_connector": "*",
+        "drupal/acquia_telemetry-acquia_telemetry": "1.0.0-alpha6",
```

Comment:
I notice a lot of modules were added to composer.json, is there a reason? Are these subdependencies that needed to be pinned to a specific version?

Reply:
These are added by the upgrade to [Lightning_core 6.0](https://www.drupal.org/project/lightning_core/releases/6.0.0). But let me take a look at the version pinning. Should be able to make it more flexible.

Reply:
Most of the new entries are removed. For the remaining ones, I removed the version pinning.